backup/backup.go

```diff
@@ -98,6 +119,16 @@ func (opts *Options) flushData(ctx context.Context, client *mongo.Client) error
 			catcher.Add(err)
 			break
 		}
+
+		if opts.EnableLogging && seen&1000 == 0 {
```

Comment:
`seen&1000`?

Reply:
only log every thousandth document dumped

Reply:
I think you mean `seen % 1000`